build-system/tasks/bundle-size/APPROVERS.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dist/v0.js": {
+    "approvers": ["ampproject/wg-performance", "ampproject/wg-runtime"],
+    "threshold": 0.1
```

Comment:
For clarity, would it make sense to call this `thresholdKb`?

Reply:
I don't like this name though :(

Reply:
Yeah, `thresholdKb` sounds goofy. I was thinking JS variable names like `delay_ms`. Documentation is good enough here.